### ISSUE TITLE
No longer works with i18next update (13.0.0) :/

### ISSUE THREAD

**Issue by CorentinHeroux** [1c7/react-native-langauge-hot-reload-example#1]
Everything is said in the title.

**Comment by 1c7** [comment]
Then I am sorry it's not working.
I have no intention to maintain this repo anymore.
please looking for other solution that works

**Comment by CorentinHeroux** [comment]
yes I think, it's just to prevent possible mistakes for future people :)

**Comment by 1c7** [comment]
Then let me leave this issue open.
